homeassistant/components/wiz/light.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, wiz_data: WizData, name: str) -> None:
             )
         if features.color_tmp:
             self._attr_supported_color_modes.add(ColorMode.COLOR_TEMP)
-        if not self._attr_supported_color_modes and features.brightness:
+        if not self._attr_supported_color_modes:
```

Comment:
The check is modified to match the check when calculating the active color mode:
https://github.com/home-assistant/core/blob/a08e5f3238073ab673b3d33dcd067deee931d6ae/homeassistant/components/wiz/light.py#L120-L121

Alternatively, there should be a fallback to `ColorMode.ONOFF` it not all wiz lights support brightness